### DISCUSSION TITLE
All seems ok, but device is not active in the app

### DISCUSSION THREAD

**Discussion by uniquwy** [eusonlito/GPS-Tracker#82] [Q&A] [answered]
Hi and thank you for great product!

After a few days of installation on the Debian 11, at least I see working interface of your app.
But after adding of the device ST-901L-4G and configuring it by SMS, I still not see any information in the application. 

Server configured as "h02" and in the log contains a lot of strings like these:
```[2024-10-27T23:58:37+00:00] 2491710096272252262610245025936000030303430c000000fbfffdff0000e7f40000000000ff036880c42158
[2024-10-27T23:58:39+00:00] 2491710096272252262610245025936000030303430c000000fbfffdff0000e7f40000000000ff036880c42159
[2024-10-27T23:58:41+00:00] 2491710096272252262610245025936000030303430c000000fbfffdff0000e7f40000000000ff036880c4215a
[2024-10-27T23:58:43+00:00] *HQ,9171009627,V5,235842,A,5030.9395,N,3014.6321,E,0.00,0,271024,FFFFFDFF,255,3,6880,c421,0#
[2024-10-27T23:58:45+00:00] 2491710096272252262610245025936000030303430c000000fbfffdff0000e7f40000000000ff036880c4215b
[2024-10-27T23:58:47+00:00] 2491710096272358462710245030939600030146320e000000fffffdff0000e7f40000000000ff036880c4215c
[2024-10-27T23:58:49+00:00] 2491710096272252262610245025936000030303430c000000fbfffdff0000e7f40000000000ff036880c4215d
[2024-10-27T23:58:51+00:00] 2491710096272252262610245025936000030303430c000000fbfffdff0000e7f40000000000ff036880c4215e
[2024-10-27T23:58:53+00:00] 2491710096272252262610245025936000030303430c000000fbfffdff0000e7f40000000000ff036880c4215f
```

As I understand, server successfully receives data from the tracker, but there is messages of two different protocols (why?). The "*HQ,..." looks like Sinotrack, but such lines are rare, one time from a dozen. What protocol used in the another, more numerous lines? How make app to work in this case?

Thanks.

**Comment by eusonlito** [comment]
Ok, let me see :)

**Comment by eusonlito** [comment]
Can you send me a more large log in movement? I need to verify some values. Thanks!

**Reply by uniquwy** [reply]
> Can you send me a more large log in movement? I need to verify some values. Thanks!

Of course.
[server.zip](https://github.com/user-attachments/files/17556361/server.zip)

**Reply by eusonlito** [reply, marked as answer]
Updated source, please upgrade your installation with `./composer deploy`

**Reply by uniquwy** [reply]
> Updated source, please upgrade your installation with `./composer deploy`

Updated. Now last device connection time is shown and updates every few seconds:
![screenshot](https://github.com/user-attachments/assets/4fb209ed-1785-4f1d-a7ee-5d88cd34f249)

But coordinates are not displayed and table **position** still empty.

Server protocol parser's log also is empty:
![screenshot](https://github.com/user-attachments/assets/efdf7770-2583-4ca1-a6e3-1f74698dc7d2)

**Reply by eusonlito** [reply]
These panel is always empty. Can you paste the -debug.log and check the result?

**Reply by uniquwy** [reply]
Seems to ok:

![screenshot](https://github.com/user-attachments/assets/84f12fd1-a8ad-4619-be09-691ae2721f81)

**Reply by eusonlito** [reply]
And you haven't any point? Let me check in my local environment.

**Reply by uniquwy** [reply]
You can to log in:
**************

**Reply by eusonlito** [reply]
Ok, I can create the trips without problem using your logs:

![image](https://github.com/user-attachments/assets/a7f31244-179b-407b-bae3-b28aa93e51a6)

I will check your laravel logs.

**Reply by uniquwy** [reply]
Bad logs. I will try to solve myself.

**Reply by uniquwy** [reply]
yes. at least it worked. details will a soon

**Reply by uniquwy** [reply]
Sorry, in process of experiments I got error 405:

![screenshot](https://github.com/user-attachments/assets/264b9327-4791-4d04-96a7-0d6909aa4e1f)

And even restoring of the app folder backup does not help. Can you say how to fix it?

**Reply by eusonlito** [reply]
This is a common Laravel error when not installed as domain root. Try to do a `./composer deploy`.

**Reply by uniquwy** [reply]
I don't know why, but application works only if **/index.php** specified in the URL. Request to the app root folder always return error 405 as above.

But thank you for assistance!
My solution is posted [here](https://github.com/eusonlito/GPS-Tracker/discussions/83) to your notice.

Good luck!
